File: entrada/archivo2.c
#include <stdio.h>
#include "archivo5.h"
int main() {
    char operador;
    double num1, num2, resultado;

    printf("Introduce el primer número: ");
    scanf("%lf", &num1);

    printf("Introduce el operador (+, -, *, /): ");
    scanf(" %c", &operador);

    printf("Introduce el segundo número: ");
    scanf("%lf", &num2);

    switch (operador) {
        case '+':
            resultado = num1 + num2;
            break;
        case '-':
            resultado = num1 - num2;
            break;
        case '*':
            resultado = num1 * num2;
            break;
        case '/':
            if (num2 != 0) {
                resultado = num1 / num2;
            } else {
                printf("Error: División por cero no permitida.\n");
                return 1;
            }
            break;
        default:
            printf("Operador no válido.\n");
            return 1;
    }

    printf("Resultado: %.2f\n", resultado);
    return 0;
}

File: salida/archivo2.rs
fn () main-> i32 {
	let operador:char ;
	
	
	let num1:f64 ;
	
	num2:f64 ;
	
	resultado;
	
	
	print!("Introduce el primer número: ");
	let mut input = String::new();
	io::stdin()
	.read_line(&mut input)
	.expect("Failed to read line");
	print!("Introduce el operador (+, -, *, /): ");
	let mut input = String::new();
	io::stdin()
	.read_line(&mut input)
	.expect("Failed to read line");
	print!("Introduce el segundo número: ");
	let mut input = String::new();
	io::stdin()
	.read_line(&mut input)
	.expect("Failed to read line");
	match (operador){
		"+" => {
			resultado:=num1 + num2;
			
			
			break;
			
			
		}, "-" => {
			resultado:=num1 - num2;
			
			
			break;
			
			
		}, "*" => {
			resultado:=num1 * num2;
			
			
			break;
			
			
		}, "/" => {
			if (num2 != 0){
				resultado:=num1 / num2;
				
				
			}
			else{
					print!("Error: División por cero no permitida.\n");
				1}
			
			break;
			
			
		}, _ => {
			print!("Operador no válido.\n");
			1
		}}
	
	print!("Resultado: %.2f\n{}",resultado);
	0
}

